scenarios/aws/microVMs/microvms/resources/resources.go

```diff
@@ -22,8 +23,23 @@ const (
 	PoolPath      = "poolPath"
 	Nvram         = "nvram"
 	Efi           = "efi"
+	fileConsole   = "file"
+	ptyConsole    = "pty"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should console type be a subset of `string` that allows only `"file|pty"` values ?

Reply:
How would I implement this?

Reply:
Using a go "enum":
```
type ConsoleType string
const (
    FileConsole ConsoleType = "file"
    PTYConsole ConsoleType = "pty"
)
```